app/Controllers/Http/TagController.js

```diff
@@ -11,36 +10,29 @@ class TagController {
   /**
    * Show a list of all tags.
    * GET tags
-   *
-   * @param {object} ctx
-   * @param {Request} ctx.request
-   * @param {Response} ctx.response
-   * @param {View} ctx.view
    */
-  async index ({ request, response, view }) {
+  async index({ request }) {
```

Comment:
This `request` param is not being used, we can remove that I think.

Reply:
Yes, request is being used as showed below:

```suggestion
async store({ request }) {
    const data = request.only(["name"]);

    const tag = await Tag.create(data);

    return tag;
  }```

Reply:
I don't see `request` being used within the `index` function, not the `store`.
```javascript
async index ({ request }) {
  const tags = await Tag.query().fetch();
  return tags;
}
```